disdrodb/L0/readers/AUSTRALIA/MELBOURNE_2007_OTT.py

```diff
@@ -332,4 +268,4 @@ def df_sanitizer_fun(df, lazy=False):
 
 
 if __name__ == "__main__":
-    main()
+    reader()
```

Comment:
`if __name__ == "__main__":` and code below to remove !
This comment applies to all readers

Reply:
ok